koku/api/iam/models.py

```diff
@@ -76,9 +76,14 @@ class UserPreference(models.Model):
                             unique=True, null=False)
     user = models.ForeignKey('User', null=False, on_delete=models.CASCADE)
     preference = JSONField(default=dict)
+    name = models.CharField(max_length=255, null=False, default=uuid4)
+    description = models.CharField(max_length=255, null=True)
+
+    class Meta:
+        ordering = ['preference']
```

Comment:
How does this choose to order items since its a dictionary? Does `name` make more sense?

Reply:
I added `name` late in development, long after I'd set the ordering. Now that `name` is in the schema, yes, that's a more sensible thing to order by.

I just needed to set this to something other than `id` or `uuid` because of how I'd written the tests for the defaults.